temporalio/client.py

```diff
@@ -5502,8 +5502,12 @@ async def create_schedule(self, input: CreateScheduleInput) -> ScheduleHandle:
                     overlap_policy=temporalio.api.enums.v1.ScheduleOverlapPolicy.ValueType(
                         input.schedule.policy.overlap
                     ),
-                ),
-                backfill_request=[b._to_proto() for b in input.backfill],
+                )
+                if input.trigger_immediately
+                else None,
+                backfill_request=[b._to_proto() for b in input.backfill]
+                if input.backfill
+                else None,
```

Comment:
(Also usual Python formatting style is to place parens around `xxx if cond else yyy` expressions when used as arguments so that `ruff` indents the expression:

```python
            initial_patch = temporalio.api.schedule.v1.SchedulePatch(
                trigger_immediately=(
                    temporalio.api.schedule.v1.TriggerImmediatelyRequest(
                        overlap_policy=temporalio.api.enums.v1.ScheduleOverlapPolicy.ValueType(
                            input.schedule.policy.overlap
                        ),
                    )
                    if input.trigger_immediately
                    else None
                ),
                backfill_request=(
                    [b._to_proto() for b in input.backfill] if input.backfill else None
                ),
            )
```